src/client/ds/object_meta.h

```diff
@@ -156,6 +156,14 @@ class ObjectMeta {
    */
   void AddKeyValue(const std::string& key, const std::string& value);
 
+  /**
+   * @brief Add a string value entry to the metadata from the environment
+   * variable.
+   *
+   * @param key The name of metadata entry.
+   */
+  void AddKeyValueFromEnv(const std::string& key);
```

Comment:
I think you mean add a key-value pair to the metadata where the value is taken from environment variable? looks strange that force bind the key to the environnment variable name in the metadata.

BTW I would like to not adding such method into the vineyard client core and our `AddKeyValue*` method should already have handled necessary cases.